README.md

```diff
@@ -0,0 +1,10 @@
+# health-steps-remote
+
+Basic app that can send steps data to a remote server. Default behavior is to
+upload the last 15 minutes worth of minute data after that interval, but can be
+configured in `src/config.h`.
+
+The server receiving the data is selected with the `NODE_SERVER_URL` variable in
+`src/js/pebble-js- app.js`, and in this example is a simple Node server with a
+function (`onNewData()`) broken out to allow use of the received data array as
+preferred.
```

Comment:
Would be worth just explaining how to setup, configure and launch the server?

1. Launch the server:
$ node app.js
2. Edit the NODE_SERVER_URL in pebble-js-app.js to point to your computer IP.
3. Build the Pebble application and install the PBW file on your Pebble

Please note: your mobile phone must have network access to the computer running the server.